### DISCUSSION TITLE
Can't connect my github account to terminal

### DISCUSSION THREAD

**Discussion by UtkuKaanArasan** [community/community#23568] [New to GitHub]
<p>So,my problem is when i commit and push my code to my repository it shows the username that i typed not my account. How can i fix this.<br>
<img src="https://user-images.githubusercontent.com/76698597/181077953-e0142b0d-9f9d-4cb2-9acf-a08ae225ed5c.png" alt="Ekran görüntüsü 2022-01-23 180637" data-base62-sha1="wYwaKGvZBa0JAr09lwP43CNbhZD" width="353" height="152"></p>
<p>Note: This is my first time here so if i did something wrong please let me know.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>When you make a commit, Git records the locally set author name and email address in the commit. When you push the commits to GitHub, GitHub uses the email address to connect commits to accounts. Check if the email address you have configured is correct with:</p>
<pre><code class="lang-auto">git config user.email
</code></pre>
<p>And if not <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git">correct it</a>. If the email address is correct but not on your account, <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/adding-an-email-address-to-your-github-account">add it</a>.</p>
<p>Afterwards you can go back and <a href="https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History">edit the commit history</a> to change the address.</p>
